README.md

```diff
@@ -1 +1,32 @@
-# Portfolio
\ No newline at end of file
+# My Portfolio
+
+> This is my portfolio describing what I am capable of doing and what value I can bring in a team.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Authors
+
+👤 **Samuel Fleice**
+
+- GitHub: [@samWfelcie](https://github.com/samWfelice)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@samwfelice](https://twitter.com/samwfelice)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samfelice/)
+

```

Comment:
- [ ] [OPTIONAL] I think it would be cooler 👍 if you remove the second author 👨 from authors (since it is redundant)